Source/options.js

```diff
@@ -453,5 +453,39 @@ $(function() {
     save('ngramMax', 'Maximum word sequence length saved');
   }
 
+  // Listener to update blacklist from context menu
+  chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
```

Comment:
@Ceilican, if I move the functionality into background.js, I will have to implement the entire functionality of **saveBulk** from option.js. Don't you think that will lead to unnecessary duplication? Right now the message gets queued and is handled as soon as the options page is opened. What do you think?